inventory.yaml

```diff
@@ -9,9 +9,6 @@ images:
     vars:
       context: .
       template_context: scripts/dev/templates/agent
-      # Default values but overwritten in pipeline.py
```

Comment:
Did these default values caused troubles in your setup ?

Reply:
What did exactly happen? What was the error?

Reply:
isn't that because we hardcoded the tools_distr and agent_distro?
But wouldn't it work if you still keep the defaults? 😕 

Nevertheless, works for me!

Reply:
yes, we actually had a bug! We pulled that fix out https://github.com/mongodb/mongodb-kubernetes-operator/pull/1596